wps-cloudformation-template.yaml

```diff
@@ -358,6 +369,30 @@ Resources:
           ResourcePath: /*
       RestApiId: !Ref WPSApi
       StageName: LATEST
+  WPSApiDomainName:
+    Type: 'AWS::ApiGateway::DomainName'
+    Condition: CreateCustomDomainName
+    Properties:
+      CertificateArn: !Ref acmCertificate
+      DomainName: !Ref domainName
+  DomainToAPIMapping:
+    Type: 'AWS::ApiGateway::BasePathMapping'
+    Condition: CreateCustomDomainName
+    Properties:
+      DomainName: !Ref WPSApiDomainName
+      RestApiId: !Ref WPSApi
+      Stage: LATEST
+  Route53RecordSet:
+    Type: 'AWS::Route53::RecordSet'
+    Condition: CreateCustomDomainName
+    Properties:
+      AliasTarget:
+        DNSName:
+          Fn::GetAtt: [ 'WPSApiDomainName', 'DistributionDomainName' ]
+        HostedZoneId: 'Z2FDTNDATAQYW2'
```

Comment:
As discussed, lets move this static value to the mapping.

Reply:
This is static across all regions and all accounts, so doesn't make sense to include in mapping.